pkg/ssh/exec.go

```diff
@@ -17,16 +17,18 @@ import (
 	"context"
 	"fmt"
 	"io"
+	"os"
 	"strings"
 	"time"
 
 	"github.com/okteto/okteto/pkg/log"
 	"golang.org/x/crypto/ssh"
+	"golang.org/x/crypto/ssh/agent"
 	"golang.org/x/crypto/ssh/terminal"
 )
 
 // Exec executes the command over SSH
-func Exec(ctx context.Context, remotePort int, tty bool, inR io.Reader, outW, errW io.Writer, command []string) error {
+func Exec(ctx context.Context, remotePort int, tty bool, agentFwd bool, inR io.Reader, outW, errW io.Writer, command []string) error {
```

Comment:
Since both callers of ssh.Exec are passing true, I think it's better to not even have the agentFwd argument here.